modules/deepintentDpesIdSystem.md

```diff
@@ -0,0 +1,42 @@
+# Deepintent DPES ID
+
+The Deepintent Id  is a shared, healthcare identifier which helps publisher in absence of the 3rd Party cookie matching. This lets publishers set and bid with healthcare identity . Deepintent lets users protect their privacy through advertising value chain, where Healthcare identity when setting the identity takes in consideration of users choices, as well as when passing identity on the cookie itself privacy consent strings are checked. The healthcare identity when set is not stored on Deepintent's servers but is stored on users browsers itself. User can still opt out of the ads by https://option.deepintent.com/adchoices
+
+## Deepintent DPES ID Registration
+
+The Deepintent DPES ID is free to use, but requires a simple registration with Deepintent. Please contact your sales rep to get started
+
+
+## Deepintent DPES ID Configuration
```

Comment:
as mentioned in the docs PR, it seems to be required that publishers have additional Deepintent code on their page to retrieve and store the DPES ID in order for the user id module to pick it up. Assuming that's the case, I think that should be clearly documented here somewhere

Reply:
@smenzer : as suggested I added more details about the Deepintent registration and a tag code that publisher needs to integrate to get started with this user module, can we merge this now ?